### PR TITLE
DOC-2817: Update CA doc for nav enhancements

### PR DESCRIPTION
**Pull request by catong**
DOC-2817
This PR updates the B&R guide(s) to reflect navigation enhancements, including the introduction of Next/Prev buttons in the LMS.
### Reviewers

Possible roles follow. PR submitter checks the boxes after each reviewer
finishes and gives :+1:. 
- [x] Subject matter expert: @nasthagiri 
- [x] Doc team review (sanity check/copy edit/dev edit): @lamagnifica @srpearce @pdesjardins 

FYI: @jaakana
### Testing
- [x] Ran ./run_tests.sh without warnings or errors
### Post-review
- [x] Add description to release notes task as a comment
- [x] Squash commits

### CODE REVIEW

File: en_us/shared/developing_course/course_outline.rst [hunk 1]
```diff
@@ -45,9 +45,9 @@ You must :ref:`create the first section<Create a Section>`.
 Example of a Developed Course in the Outline
```
Comment (lamagnifica) [inline en_us/shared/developing_course/course_outline.rst]: if you're willing to take the opportunity, I would love to suggest changing this heading to something more along the lines of, "Course Outline Example" or "Understanding the Course Outline"...

File: en_us/shared/set_up_course/creating_new_course.rst [hunk 2]
```diff
@@ -78,12 +78,13 @@ To create a course, follow these steps.
 #. Enter the :ref:`identifying course information<Identifying the Course>`, and
    then select **Create**.
 
-  .. image:: ../../../shared/images/new_course_info.png
+   .. image:: ../../../shared/images/new_course_info.png
      :width: 600
      :alt: Image of the course creation page in Studio.
```
Comment (lamagnifica) [inline en_us/shared/set_up_course/creating_new_course.rst]: again, outside of scope but nice to have: replace current alt text with something like "All of the fields for creating a course in Studio: Course Name, Organization, Course Number, and Course Run.

**Comment by lamagnifica** [comment]
:+1: my only suggestions are for surrounding text :-/

**Comment by nasthagiri** [comment]
Question: For the changes in the `outline-callouts.png` file, are the representative numbers defined somewhere?

**Comment by catong** [comment]
@nasthagiri Yes, there's a numbered list directly under the graphic that has:
1. Sections
2. Subsections
3. Units

**Comment by nasthagiri** [comment]
:+1:  Thanks.

File: en_us/shared/developing_course/course_outline.rst (hunk 1 above)
Comment (catong) [inline en_us/shared/developing_course/course_outline.rst]: Done

File: en_us/shared/set_up_course/creating_new_course.rst (hunk 2 above)
Comment (catong) [inline en_us/shared/set_up_course/creating_new_course.rst]: Done

**Comment by catong** [comment]
Merging https://github.com/edx/edx-documentation/pull/960

**Pull request merged**